### PR TITLE
Setup Sequelize ORM

### PR DESCRIPTION
**Pull request by max-wel**
#### What does this PR achieve?
- adds migrations
- adds models
- adds seeders
#### Manual Testing Steps
Create ```.env file``` and add **DB** credentials
Run ```npx sequelize db:migrate```
Run ```npx sequelize db:seed:all```
#### Relevant Trello Tickets
- [Ticket](https://trello.com/c/Q0ZaEbOs)

### CODE REVIEW

File: src/database/seeders/20200109111618-demo-role.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      [
+        {
+          id: 'e4af43af-ca38-426b-a8ec-65a52a5a45fc',
+          name: 'Admin',
+          description: 'The boss himself',
+        },
+      ],
+      {},
+    );
+  },
```
Comment (chukwuemekachm) [inline src/database/seeders/20200109111618-demo-role.js]: Can you add more roles such as the `Owner`, `Volunteer`, `Super Admin`, and `Manager`?

File: src/database/seeders/20200109111635-demo-membership.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert(
+      'Memberships',
+      [
+        {
+          userId: '40b6c719-68b2-4097-95ea-6fe230784e29',
+          orgId: '546c692b-0cd5-4cea-b391-2a481c8c1975',
+          roleId: 'e4af43af-ca38-426b-a8ec-65a52a5a45fc',
+        },
+      ],
```
Comment (chukwuemekachm) [inline src/database/seeders/20200109111635-demo-membership.js]: 👍🏽

File: src/database/seeders/20200109111519-demo-user.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        {
+          id: '40b6c719-68b2-4097-95ea-6fe230784e29',
+          name: 'John Doe',
+          email: 'john.doe@gmail.com',
+          username: 'john-doe',
+          password: 'pass',
+          verified: false,
+        },
+      ],
+      {},
```
Comment (chukwuemekachm) [inline src/database/seeders/20200109111519-demo-user.js:25]: Can we seed one more user here, who will be a `Super Admin` as the above user is already linked to an organization?

File: src/server/repositories/user.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import db from '../../database/models/index.js';
+
+const { User } = db;
+
+export default class UserRepository {
+  
+}
```
Comment (chukwuemekachm) [inline src/server/repositories/user.ts:7]: 👍🏽

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Nice work @max-wel , 🙂
Just a couple of things below that I will like to be addressed.

File: src/database/seeders/20200109111618-demo-role.js (hunk 1 above)
Comment (max-wel) [reply src/database/seeders/20200109111618-demo-role.js]: Sure

**Review by chukwuemekachm — APPROVED** [review]

**Pull request merged**